### DISCUSSION TITLE
Kindly publish to fdroid/Izzy's repo

### DISCUSSION THREAD

**Discussion by kryoqa** [enrique-lozano/Monekin#262] [Ideas / New features]
I recently found this app after searching for an alternative to my daily app which was archived. So far I am liking Monekin and want to thank the devs and contributors for making this. 

However, I (and many others) don't really use Play Services and use fdroid to get apps and updates. Will it be possible to publish Monekin to Fdroid/Izzy's repo ?
(I am using github releases for installing Monekin)

**Comment by enrique-lozano** [comment]
Hello! Thanks for using the app and for you appreciation words. 

Yeah, I think that would be great! Will have to take some time investigating how can I upload an app to fdroid, but I think it shouldn't be complicated. Right now each release it's quite time consuming and I'm a little frightened to have one more step to do, but certainly it will be a nice addition to this app.

Will notice you with new advances in this regard occur!

**Reply by kryoqa** [reply]
Thank you! 

If it's any help, I was able to find these : 

https://apt.izzysoft.de/fdroid/index/info

https://gitlab.com/IzzyOnDroid/repo

Apps are taken mostly from their own repos as far as I know, but there is an inclusion policy for certain trackers and packages. I wish I could be of more help here, but I don't know much about Android development.

If adding this becomes more work, then it's understandable. It'll be fine as long as an apk exists on Github releases ❤️
